src/servant/tt_002/faerie.c

```diff
@@ -461,6 +456,10 @@ void CheckForValidAbility(Entity* self) {
     }
 }
 
+#ifdef VERSION_PC
+extern u16 g_FaerieClut[64];
```

Comment:
Why is this needed? The storage doesn't exist elsewhere?

Reply:
I did it this way because that was how the bat and the ghost and bat did it, but yeah, you're right, we can just do that in the actual extern declaration of the array.  The PSX version is fine with it as well.  I made that update in the bat and ghost as well to get rid of that ifdef